compiler/semstmts.nim

```diff
@@ -877,7 +877,7 @@ proc semCase(c: PContext, n: PNode; flags: TExprFlags): PNode =
   pushCaseContext(c, n)
   n.sons[0] = semExprWithType(c, n.sons[0])
   var chckCovered = false
-  var covered: BiggestInt = 0
+  var covered: Int128 = toInt128(0)
```

Comment:
I'm curious, wouldn't using uint64 instead of BiggestInt work too? Will `covered` ever be negative?

Reply:
It would work yes, but we also will support ``high(uint64)`` so we need an integer type internally that is bigger than every exposed integer type.